### PR TITLE
Digest-Update feature

### PR DESCRIPTION
**Pull request by inklin**
- Move sendAs field from Digest-Updates to Digest
- Drop table Digest-View-Settings
- Create table Digest-Recipient-Settings (for a person's settings: when and how they want to receive their digests)
- Add function: takes an update, and creates a Digest-Update for each person on the update author's contacts list. (Looks at each receiver's settings, unsent digest, and creates a Digest-Update to connect the update with the receiver's Digest)

### CODE REVIEW

File: lib/handlers/digest-update.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+          if (digests.length === 0) {
+
+            // Create a new Digest
+            models.Digest.create({
+              PersonId: PersonId,
+              sendAs: viewSetting
+            }).then(function(digest) {
+
+              // Create a Digest_Update
```
Comment (ggrochow) [inline lib/handlers/digest-update.js]: I think one of these create digest updates is redundant, you could store a digest variable and create the update after either finding or creating it

File: lib/handlers/digest-update.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+            }, function(err) {
+              reply(Boom.badRequest('Failed to create Digest', err));
+            });
+
+          } else {
+            var digest = digests[0];
+
+            // Create Digest_Update
+            models.DigestUpdate.create({
```
Comment (ggrochow) [inline lib/handlers/digest-update.js]: One of these

File: lib/handlers/digest-update.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    where: { AccountId: AccountId }
+  }).then(function(contacts) {
+
+    // Grab each Person's DigestRecipientSettings
+    contacts.forEach(function(contact) {
+
+      var PersonId = contact.PersonId;
+
+      models.DigestRecipientSettings.findAll({
+        where: { PersonId: PersonId }
```
Comment (ggrochow) [inline lib/handlers/digest-update.js]: doesn't this only need to happen if a digest doesn't exist

Comment (inklin) [inline lib/handlers/digest-update.js]: You mean when you get an update, then look for the Digest right away?

Comment (ggrochow) [inline lib/handlers/digest-update.js]: Sorry on my phone might have pointed at wrong line. 
I meant getting the settings

**Comment by livfwds** [comment]
looks good!

**Pull request merged**